service/interface.go

```diff
@@ -15,7 +15,7 @@ import(
 )
 type Manager interface{
 	CreateUser(*domain.User) (string,error)
```

Comment:
although, the parameter's names are optional here,
But if you add those, it will increase readability

Reply:
Done ! Please check